src/app/[locale]/page.tsx

```diff
@@ -4,13 +4,19 @@ import BodyWrapper from '../components/BodyWrapper';
 import Navbar from '../components/Navbar';
 import Header from '../components/Header';
 import Services from '../components/Services';
+import Process from '../components/Process';
 
 const Home = () => {
   return (
-    <BodyWrapper>
-      <Navbar />
-      <Header />
-      <Services />
+    <>
+      <BodyWrapper>
+        <Navbar />
+        <Header />
+        <Services />
+      </BodyWrapper>
+
+      <Process />
```

Comment:
I see that you couldn't put it inside the bodyWrapper but maybe we should find a way to apply the background only for this component no?